CHANGELOG.md

```diff
@@ -24,7 +24,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Support for nesting objects in properties, invokables, and signals with `*mut T`
 - Allow for marking signals as existing in the base class
 - Support for conversions to types in third-party crates: `bytes`, `chrono`, `http`, `rgb`, `time`, `url`
-- Add several QoL functions to builtin cxx-qt-lib types, including Default constructors, string formatting, std::cmp order and operators
+- Add several QoL functions to builtin cxx-qt-lib types, including `Default` constructors, string formatting, `std::cmp` order and operators
```

Comment:
could you spell out QoL here, haven't encountered the acronym before (assuming `Quality of Life`).

Reply:
I was reminded of QoS (Quality of Service) from networking when i first saw it :-)

Reply:
Aaaah, that's what it reminded me of xD